nvidia-bert/ort_addon/ort_supplement/ort_supplement.py

```diff
@@ -3,17 +3,23 @@
 
 import os
 import onnx
-from onnxruntime.capi.ort_trainer import ORTTrainer, IODescription, ModelDescription
-from onnxruntime.capi.ort_trainer import LossScaler
 import torch
-from ort_supplement.azureml_adapter import set_environment_variables_for_nccl_backend, get_local_rank, get_local_size, get_global_size, get_world_size, get_world_rank 
+
+# onnxruntime training API
+from onnxruntime.training import orttrainer, optim, amp as ort_amp
```

Comment:
As this is an education notebook, import `ORTTrainer` and `ORTTrainerOptions` directly instead of `ortrainer`
Importing an intermediate namespace can be confusing, although it does save some typing :)

Reply:
I agree. Though I intend an additional two PRs:
1. Update the modeling.py with dense sequence optimization.
2. Refactor the script to eliminate Nvidia code and simplify it.

So I think making the script a single .py file could occur on refactor PR.

Reply:
Oh, I think I misread your comment. OK, let me change that.